crates/bevy_gizmos/src/lib.rs

```diff
@@ -476,59 +463,92 @@ struct LineGizmoUniform {
     _padding: f32,
 }
 
-/// A gizmo asset that represents a line.
+/// A collection of lines.
+///
+/// Has the same line drawing API as [`Gizmos`](crate::gizmos::Gizmos).
 #[derive(Asset, Debug, Clone, TypePath)]
-pub struct LineGizmo {
-    /// Positions of the gizmo's vertices
-    pub positions: Vec<Vec3>,
-    /// Colors of the gizmo's vertices
-    pub colors: Vec<LinearRgba>,
-    /// Whether this gizmo's topology is a line-strip or line-list
-    pub strip: bool,
-    /// Whether this gizmo should draw line joints. This is only applicable if the gizmo's topology is line-strip.
-    pub joints: GizmoLineJoint,
-    /// The type of the gizmo's configuration group
-    pub config_ty: TypeId,
+pub struct LineGizmoAsset {
```

Comment:
I'm wondering: What is the best way to change a entity retained gizmos? Like position for example
The only way now is by replacing this handle in the LineGizmos component, right?